libdnf/rpm/solv/goal_private.hpp

```diff
@@ -59,6 +60,11 @@ class GoalPrivate {
     void add_upgrade(libdnf::solv::IdQueue & queue, bool best, bool clean_deps);
     void add_distro_sync(libdnf::solv::IdQueue & queue, bool strict, bool best, bool clean_deps);
 
+    void add_group(
```

Comment:
I am curious what the methods is supposed to do. Please could you add a doc string?